kit/src/components/indicator/mod.rs

```diff
@@ -74,11 +74,18 @@ pub struct Props {
 pub fn Indicator(cx: Scope<Props>) -> Element {
     let icon = cx.props.platform.to_icon();
     let status = cx.props.status.to_string();
+    let mut circle_icon_class = "".to_owned();
```

Comment:
Instead of having a specific class, we should maybe just make a "filled" class. Apply the class for now, only when the circle is used.